index.html

```diff
@@ -28,11 +28,11 @@
         </div>
         <div class="row" id="questionOne">
             <div class="col-sm-1"></div>
-            <div class="col-sm-10" id="questions">
+            <div class="col-sm-10 questions" id="question_1">
                 <p>Which one of these characters does Jerry famously despise the most?</p>
```

Comment:
Do you see why making that a class and adding a unique id makes sense? It lets us grab the div for question 1 explicitly.